src/Php54/README.md

```diff
@@ -1,17 +0,0 @@
-Symfony Polyfill / Php54
```

Comment:
We should keep a nice README even for the now empty packages. The README is rendered on the split repos on GitHub as well as on the packagist page of the package.

Reply:
readme files are back